packages/core/types/common.d.ts

```diff
@@ -16,6 +16,7 @@ export interface IConfig {
   user?: object | null;
   metaData?: object | null;
   logger?: ILogger | null;
+  filters?: string[];
```

Comment:
Should this be `filters?: (string|RegExp)[];`?

Reply:
Yeah, or it needs to allow a mixture which I'm not sure that does? I'll fix it, thanks!

Reply:
your example was correct